apps/src/code-studio/components/libraries/libraryParser.js

```diff
@@ -145,27 +145,38 @@ export function createLibraryJson(
 }
 
 /**
- * Removes all illegal characters from a library's name and insures it starts
- * with a capitalized letter.
+ * Creates a suggested name for a library by removing all illegal characters
+ * and ensuring is starts with a capitol letter.
+ * @param {string} libraryName The name of the library
+ * @returns {string} The name of the library without any illegal characters.
+ */
+export function suggestName(libraryName) {
+  let suggestedName = sanitizeName(libraryName);
+  if (suggestedName.length === 0 || !isNaN(suggestedName.charAt(0))) {
+    suggestedName = 'Lib' + suggestedName;
```

Comment:
Is this okay for non-English libraries?

Reply:
Hmmm great question. I'll circle back with the curriculum & prod teams on that one.